src/index.js

```diff
@@ -18,9 +18,20 @@ import 'babel-polyfill';
 import fs from 'fs';
 import path from 'path';
 import vorpal from 'vorpal';
-import { version } from '../package.json';
+import semver from 'semver';
+import { version, engines } from '../package.json';
 import config from './utils/config';
 
+const nodeVersionError = (required, actual) =>
+	chalk.red(`
+Error: Wanted node version ${required} but has ${actual}
```

Comment:
How about `Error: ${name} requires Node.js version ${required}, but was started with ${actual}.`?

Where `name` comes from the user's config file.

Reply:
I like this!

Reply:
But I think we shouldn't access `name` in the `./bin/lisky`

Reply:
it’s not really good idea to check config file from the `bin/lisky` since config is not created here